src/agenda.rs

```diff
@@ -432,6 +470,22 @@ pub fn prioritization<'a>() -> Box<dyn Action> {
             filters: vec![("state", "open")],
             include_labels: vec!["I-nominated", "T-compiler"],
             exclude_labels: vec![],
+            ordering: HashMap::new(),
+        },
+    });
+
+    let mut ordering = HashMap::new();
+    ordering.insert("sort", "updated");
+    ordering.insert("per_page", "5");
+
+    queries.push(QueryMap {
+        name: "top_unreviewed_prs",
+        query: github::Query {
+            kind: github::QueryKind::List,
+            filters: vec![("state", "open"), ("is", "pull-request"), ("-is", "draft")],
+            include_labels: vec!["S-waiting-on-review"],
```

Comment:
Do we want to include `T-compiler` here?

Reply:
Actually, yes, this is how everything else is done. Fixed.

Reply:
Probably not actually. Most PRs don't have a T-compiler label. PRs/issues that have FCP though *need* a label though.

Reply:
cc @pnkfelix, so we can define what to exactly do here

Reply:
I was thinking that we should leave the `T-compiler` label. This is not different from the rest of the things we do here. The working group in general check for these missing labels and add them when required. That's part of the https://forge.rust-lang.org/compiler/prioritization/procedure.html. cc @apiraino maybe we should add this as part of the procedure.

Reply:
I can see both sides of this. 

I'm okay with only putting things tagged with `T-compiler` on the agenda, at least for now.

The point here is to try to make sure we stay on top of things that need review. There's a chance that relevant PR's might not get tagged. If we see that happening, then we can revisit this.

Reply:
Well, most PRs don't end up getting tagged `T-compiler`. There are only 16 open PRs tagged and waiting on review.

I don't think filtering these is unreasonable, but we would have to be much more proactive (and in my opinion more automatic) with adding `T-compiler` to relevant PRs.

Reply:
As I've said, I think this can work in the same way the rest of the things work, which is having all the issues with `T` labels and revisiting them as per procedure to be 100% sure that there are no `S-waiting-on-review` PRs without a `T` label.
So what we need to do now is:

1. Go over all `S-waiting-on-review` PRs and tag them with the right `T` label
2. Add an entry to the procedure that ensures that we check for `S-waiting-on-review` labelled PRs that miss a `T` label and tag appropriately
3. We can now be sure that the code in the way it is in this PR would always produce the desired result

Reply:
@jyn514 yes but in the past we took bolder steps with success :) so why not this one that's easier, it's just adding a label about the right team. Remember that we've prioritized all unsound issues and all regressions before.

Reply:
I agree, we have a backlog of unlabelled PR waiting for review but I would go through the process of labelling them the same. It's a one-time annoyance but after that we Wg-prio will groom the issue tracker and keep things in better shape. hopefully.

Reply:
I just finished assigning team labels to all PRs in that set. The assignments may not be perfect, but I did the best I could, and hopefully any errors won't cause problems.